ios/Classes/BetterPlayerPlugin.m

```diff
@@ -627,6 +626,23 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
                     [player showPlayerCoverView];
                 }
             }
+        } else if([@"hidePipAndControlCenterButton" isEqualToString:call.method]) {
+            NSDictionary* dataSource = [_dataSourceDict objectForKey:[self getTextureId:player]];
+            NSMutableDictionary *mutableDataSource = [dataSource mutableCopy];
+            [mutableDataSource setObject:@(false) forKey:@"showNotification"];
+            _dataSourceDict[[self getTextureId:player]] = mutableDataSource;
+
+            if (!player.isPipMode) {
+                [player resetPipController];
+            } else {
+                [_notificationPlayer setIsDisplayPipButtons:false];
+            }
+            [[NSNotificationCenter defaultCenter] removeObserver:self];
+            [self disposeNotificationData:player];
+            [self removeCommandCenterTargetHandlers];
+            [self setRemoteCommandsNotificationNotActive];
```

Comment:
```suggestion
            [[UIApplication sharedApplication] endReceivingRemoteControlEvents];
```
In this function have disable `AVAudioSession`, I think we only need to use  `endReceivingRemoteControlEvents`. Please check it.